src/transformers/integrations/flash_paged.py

```diff
@@ -48,7 +48,7 @@ def paged_attention_forward(
         window_size: (left, right). If not (-1, -1), implements sliding window local attention.
         softcap: float. Anything > 0 activates softcapping attention.
     """
-    k, v = cache.update(k, v, module.layer_idx, cumulative_seqlens_k=cumulative_seqlens_k, **kwargs)
+    k, v = cache.update(k, v, module.layer_idx, cu_seq_lens_k=cu_seq_lens_k, **kwargs)
```

Comment:
```suggestion
    k, v = cache.update(k, v, module.layer_idx, **kwargs)
```
I don't even think we need this as kwarg. See https://github.com/huggingface/transformers/blob/a9cd0b220b093aa7ea478fb9a40a324e5d1a80d6/src/transformers/generation/continuous_batching.py#L311-L326

Reply:
I agree.

Reply:
The `**kwargs` in signification is also unused, so we can remove both.
```suggestion
    k, v = cache.update(k, v, module.layer_idx)
```

Reply:
I assume that read/write index might be hidden within the kwargs, so don't want to risk it there